apps/api/src/guards/authentication.guard.ts

```
@@ -0,0 +1,36 @@
+  ExecutionContext,
+  Injectable,
+  Logger,
+  UnauthorizedException,
+} from "@nestjs/common";
+import { Request } from "express";
+import { Observable } from "rxjs";
+
+@Injectable()
+export class AuthenticationGuard implements CanActivate {
+  // private readonly jwtService: JwtService;
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)